mmdeploy/codebase/mmdet3d/deploy/voxel_detection.py

```diff
@@ -183,7 +240,7 @@ def evaluate_outputs(model_cfg,
         if out:
             logger = get_root_logger()
             logger.info(f'\nwriting results to {out}')
-            mmcv.dump(outputs, out)
+            mmengine.dump(outputs, out)
```

Comment:
`evaluate_outputs`  method should be removed.

Reply:
`get_tensor_from_input` should be removed as well.

Reply:
done.